modules/bpm/bpm-resources/ide-bpm-menu/pom.xml

```diff
@@ -12,14 +12,14 @@
 
     <name>IDE - UI - BPM - Menu</name>
     <artifactId>dirigible-ide-ui-bpm-menu</artifactId>
-    <packaging>pom</packaging>
+    <packaging>jar</packaging>
 
     <properties>
         <profile.content.phase>generate-sources</profile.content.phase>
         <content.repository.name>ide-bpm-menu</content.repository.name>
-        <content.project.name>ide-bpm-menu</content.project.name>
+        <content.project.name>ide-bpm</content.project.name>
```

Comment:
isn't it `ide-bpm-menu` - https://github.com/dirigiblelabs/ide-bpm-menu/tree/master/ide-bpm-menu